### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Comment by woodruffw** [comment]
JFYI: You might want to consider bumping `i18n-embed-fl` to `0.3` in this patchset at well; that would avoid the need to pin the `dashmap` version to the RC that you've added :slightly_smiling_face:

File: age/Cargo.toml [hunk 1]
```diff
@@ -70,14 +70,15 @@ futures = { version = "0.3", optional = true }
 pin-project = "1"
 
 # Localization
+dashmap = "=4.0.0-rc6"
```
Comment (Roguelazer) [inline age/Cargo.toml:73]: this pin isn't needed if you bump `i18n-embed-fl` to 0.3.1

File: rage/Cargo.toml [hunk 2]
```diff
@@ -45,14 +45,14 @@ maintenance = { status = "experimental" }
 # rage and rage-keygen dependencies
 age = { version = "0.5.0", path = "../age", features = ["armor", "cli-common", "plugin"] }
 chrono = "0.4"
-console = "0.13"
+console = "0.14"
 env_logger = "0.8"
 gumdrop = "0.8"
 i18n-embed = { version = "0.9", features = ["desktop-requester", "fluent-system"] }
 i18n-embed-fl = "0.2"
```
Comment (Roguelazer) [inline rage/Cargo.toml:52]: (this one too)

**Comment by str4d** [comment]
Yeah, they released that right after I made the branch 😅

**Pull request merged**